projects/plugins/jetpack/modules/shortcodes/gravatar.php

```diff
@@ -93,16 +98,60 @@ function jetpack_gravatar_profile_shortcode( $atts ) {
 		return false;
 	}
 
+	$hashed_email = hash( 'sha256', strtolower( trim( $user->user_email ) ) );
+	$cache_key    = 'grofile-' . $hashed_email;
+	$profile      = get_transient( $cache_key );
+
+	if ( empty( $profile ) ) {
+		$profile_url = sprintf(
+			'https://secure.gravatar.com/%s.json',
+			$hashed_email
+		);
+
+		$expire        = 300;
+		$response      = wp_remote_get(
+			esc_url_raw( $profile_url ),
+			array( 'User-Agent' => 'WordPress.com Gravatar Profile Widget' )
```

Comment:
Since this is also in use in the Jetpack plugin, maybe we could say "Jetpack" instead of "WordPress.com", for folks examining their logs and trying to understand why a self-hosted site would make "WordPress.com" requests to their site?

Reply:
Makes sense, updated.